mem.go

```diff
@@ -2,12 +2,60 @@ package sqrl
 
 import (
 	"fmt"
+	"math/big"
+	"os"
 
+	"github.com/dustin/go-humanize"
+	"github.com/likexian/host-stat-go"
 	"github.com/shirou/gopsutil/mem"
 )
 
-func MemInfo() {
-	v, _ := mem.VirtualMemory()
+type Ram struct {
+	total, free, swapSpace string
+}
+
+type OS struct {
+	hostname, kernel, version string
+}
+
+func ramOSInfo() (Ram, OS) {
+	v, e := mem.VirtualMemory()
+	if e != nil {
+		fmt.Println(e)
```

Comment:
I'm not sure what you mean here. Do you mean I should have a third return value for that function? And if the error is nil, then would you still want that returned?